src/utils/redux/slices/batches.ts

```diff
@@ -65,6 +65,13 @@ export const batchesSlice = createSlice({
         batches.splice(batchIndex, 1);
       }
     },
+    removeByAccounts: (state, { payload: { pkhs } }: { payload: { pkhs: RawPkh[] } }) => {
+      Object.keys(state).forEach(networkName => {
```

Comment:
maybe Object.entries -> map -> lodash.fromPairs?

Reply:
Updated, thank you!